crates/cxx-qt/src/lib.rs

```diff
@@ -9,10 +9,13 @@
 //!
 //! See the [book](https://kdab.github.io/cxx-qt/book/) for more information.
 
+use std::ops::Deref;
+use std::pin::Pin;
 use std::{fs::File, io::Write, path::Path};
 
 mod connection;
 mod connectionguard;
+pub mod qobject;
```

Comment:
do we want developers to go `cxx_qt::qobject::QObject` or `cxx_qt::QObject` ? At the moment for cxx-qt-lib everything is in the root module, but we wondered if they should be split by core/gui/quick etc, wonder what we do here?

Reply:
I'm okay with exporting it as `cxx_qt::QObject`.
I believe this was done to go in the direction that we want cxx-qt-lib to go, which is to start using submodules more.

But that's indeed inconsistent at the moment, so let's do a top-level export and fix it if we ever get around to exporting things from cxx-qt-lib via submodules.